### DISCUSSION TITLE
Self Hosting Hasura in AWS Lambda or other cloud functions (serverless backend) ?

### DISCUSSION THREAD

**Discussion by RageshAntony** [hasura/graphql-engine#9466] [Question]
Is it possible to self host Hasura (but not databases) in AWS Lambda or  other cloud functions. That is, I am asking like serverless backend using cloud functions  ?

**Comment by adas98012** [comment]
@RageshAntony 
You should be able to self host Hasura anywhere, as long as you can deploy and run the Hasura GraphQL Engine (Docker image) as a container and can connect to your database.
We have listed some of the deployment options [here](https://hasura.io/docs/latest/deployment/deployment-guides/index/#all-deployment-guides), this may not be an exhaustive list.
